webhook/resources/backup/mutator.go

```diff
@@ -50,6 +53,35 @@ func (b *backupMutator) Create(request *admission.Request, newObj runtime.Object
 	}
 	patchOps = append(patchOps, patchOp)
 
+	backupLabels := make(map[string]string)
+	volumeName, isExist := backup.Labels[types.LonghornLabelBackupVolume]
+	if !isExist {
+		err := errors.Wrap(err, "can not find the backup volume label")
+		return nil, werror.NewInvalidError(err.Error(), "")
+	}
+	volume, err := b.ds.GetVolumeRO(volumeName)
+	if err != nil {
+		err := errors.Wrapf(err, "failed to get volume: %v", volumeName)
+		return nil, werror.NewInvalidError(err.Error(), "")
+	}
+	volumeAccessMode := longhorn.AccessModeReadWriteOnce
+	if volume.Spec.AccessMode != "" {
+		volumeAccessMode = volume.Spec.AccessMode
+	}
+	backupLabels[types.GetLonghornLabelKey(types.LonghornLabelVolumeAccessMode)] = string(volumeAccessMode)
+
+	kubeStatus, err := json.Marshal(volume.Status.KubernetesStatus)
+	if err != nil {
+		return nil, werror.NewInvalidError(errors.Wrapf(err, "failed to convert k8s status as JSON string").Error(), "")
+	}
+	backupLabels[types.KubernetesStatusLabel] = string(kubeStatus)
+
+	valueBackupLabels, err := json.Marshal(backupLabels)
+	if err != nil {
+		return nil, werror.NewInvalidError(errors.Wrapf(err, "failed to convert backup labels as JSON string").Error(), "")
```

Comment:
```suggestion
		return nil, werror.NewInvalidError(errors.Wrapf(err, "failed to convert backup labels into JSON string").Error(), "")
```

Reply:
Thanks, I will follow the suggestion to correct it.